src/test/java/com/epam/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.service;
+
+import com.epam.entity.User;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceImplTest {
+    @Mock
+    private User user;
+
+    @Test
+    void getUser() {
+        MockitoAnnotations.initMocks(this);
+        UserServiceImpl userService = new UserServiceImpl();
```

Comment:
I believe it will be good if extract this init part to another method `setUp()` with the annotation `@Before` to avoid duplicate code in the next test methods.
```
 MockitoAnnotations.initMocks(this);
 UserServiceImpl userService = new UserServiceImpl();
```